samza-core/src/main/java/org/apache/samza/operators/impl/PartialJoinOperatorImpl.java

```diff
@@ -44,4 +44,9 @@
   public void onNext(M message, MessageCollector collector, TaskCoordinator coordinator) {
     // TODO: implement PartialJoinOperatorImpl processing logic
   }
+
+  @Override
+  public void onTimer(MessageCollector collector, TaskCoordinator coordinator) {
+
```

Comment:
This class has changed quite a bit after @prateekm 's RB. Will rebase against the latest.